### PR TITLE
Mend scan

### PR DESCRIPTION
**Pull request by morri-son**
## Description

add Mend scan

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [x] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] Separate ticket for tests # (issue/pr)

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Added to documentation?

- [ ] 📜 README.md
- [x] 🙅 no documentation needed

### CODE REVIEW

File: .github/workflows/mend.config [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+# apiKey=
+ 
+# userKey is required if WhiteSource administrator has enabled "Enforce user level access" option.
+# ! In case of PIPER, apiKey may not be used in this configuration, 
+#   but set in configuration of piper. 
+#   Please look at PIPER documentation for more information.
+# ! For CoDePipes you may look at CoDePipes for more information.
+# userKey=
+   
+projectName=ocm
```
Comment (Skarlso) [inline .github/workflows/mend.config:89]:
```suggestion
projectName=ocm-controller
```

File: .github/workflows/mend_scan.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      run: curl https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar -o ./wss-unified-agent.jar
+
+    - name: Run Mend Scan 
+      run: java -jar ./wss-unified-agent.jar -c $CONFIG_FILE -wss.url $WSS_URL -apiKey $API_KEY -userKey $USER_KEY -productToken $PRODUCT_TOKEN
+      env:
+        USER_KEY: ${{ secrets.MEND_USER_KEY }}
+        PRODUCT_TOKEN: ${{ secrets.MEND_SHC_PRODUCT_TOKEN }}
+        WSS_URL: ${{ secrets.MEND_URL }}
+        API_KEY: ${{ secrets.MEND_API_TOKEN }}
+        CONFIG_FILE: './.github/workflows/mend.config'
```
Comment (Skarlso) [inline .github/workflows/mend_scan.yaml:36]: I assume these exist this time? :D

Comment (morri-son) [reply .github/workflows/mend_scan.yaml:36]: yes, they are maintained at GitHub org level :-)

File: .github/workflows/mend_scan.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    - name: Set up Java 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Setup Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: 1.19.x
```
Comment (Skarlso) [inline .github/workflows/mend_scan.yaml:24]:
```suggestion
        go-version-file: '${{ github.workspace }}/go.mod'
```

**Review by Skarlso — APPROVED** [review]

**Pull request merged**